ote/src/cljs/ote/views/transit_changes.cljs

```diff
@@ -61,21 +61,21 @@
 
 (defn change-icons [{:keys [added-routes removed-routes changed-routes no-traffic-routes]}]
   [:div.transit-change-icons (stylefy/use-style style/transit-changes-icon-row-container)
-   [:div (use-style (merge style/transit-changes-legend-icon {:width "25%"}))
+   [:div {:style {:width "25%"}}
```

Comment:
When `use-style` is used, stylefy creates a class for the style so it might be a bit more performant in some cases. Probably not a factor here since the in-line style is so short.